src/main/java/ugh/fileformats/mets/MetsModsImportExport.java

```diff
@@ -71,12 +78,30 @@
 import ugh.dl.MetadataType;
 import ugh.dl.Person;
 import ugh.dl.Prefs;
-import ugh.exceptions.DocStructHasNoTypeException;
 import ugh.exceptions.ImportException;
-import ugh.exceptions.MetadataTypeNotAllowedException;
-import ugh.exceptions.PreferencesException;
-import ugh.exceptions.ReadException;
-import ugh.exceptions.WriteException;
+
+/*******************************************************************************
```

Comment:
Why are you adding the license header again?

Reply:
Same story here.